packages/widgets/src/components/NotificationMenu/stories/StoryComponent.tsx

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from 'react';
+import { Link, MemoryRouter, Route, Routes } from 'react-router-dom';
+import { Button, ButtonGroup, ButtonGroupItem } from '@ssa-ui-kit/core';
+import { NotificationMenu } from '../NotificationMenu';
+import { MarkAllReadButton, Trigger } from './StoryContent';
+import { NotificationCardProps } from '@components/NotificationCard';
+import {
+  disableButton,
+  divideOnSubArr,
+  getMockData,
+  readAll,
+} from '../helpers';
+import { ButtonGroupCustom } from '../styles';
+
+export const StoryComponent = () => {
+  const [notifications, setNotifications] = useState<NotificationCardProps[]>(
+    [],
+  );
+  const [readNotifications, setReadNotifications] = useState<
+    NotificationCardProps[]
+  >([]);
+  const [buttonItems, setButtonItems] = useState<ButtonGroupItem[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+
+  const unreadQty = divideOnSubArr(notifications).Unread.length;
+
+  const handleClickOpen = () => () => {
+    if (!isLoading) return;
+    getMockData().then((data) => {
+      setIsLoading(false);
+      setNotifications(data.notificationItems);
+      setReadNotifications(data.notificationItems);
+      setButtonItems(data.groupButtonItems);
+    });
+  };
+
+  useEffect(() => {
+    handleClickOpen();
+    if (!unreadQty) {
+      setButtonItems(disableButton(buttonItems, notifications));
+    }
+  }, [isLoading]);
+
+  const handleClickReadAll = () => {
+    setReadNotifications(readAll(notifications));
+    setButtonItems(disableButton(buttonItems, notifications));
+  };
+
+  const handleClick = (item: Pick<ButtonGroupItem, 'id' | 'text'>) => {
+    setReadNotifications(divideOnSubArr(notifications)[item.text]);
+  };
+
+  return (
+    <MemoryRouter initialEntries={['/*']}>
```

Comment:
Do you need `initialEntries` here? The default value is `["/"]`.
https://reactrouter.com/en/main/router-components/memory-router

Reply:
It is not necessary.
Done